src/Tasks/Microsoft.Build.Tasks.csproj

```diff
@@ -961,6 +962,7 @@
     <PackageReference Include="System.Reflection.Metadata" Version="1.3.0" />
     <PackageReference Include="System.Reflection.TypeExtensions" Version="4.1.0" />
     <PackageReference Include="System.Resources.Writer" Version="4.0.0" />
+    <PackageReference Include="System.Threading.Tasks.Dataflow" Version="4.6.0" />
```

Comment:
Is this necessary? I'd expect it to be rolled into `Microsoft.NETCore.App`.

Reply:
I was mirroring what the engine core csproj does exactly to avoid different file versions colliding with each other. If the core engine upgrades it's packages this will need to be done as well